read.csv

```diff
@@ -0,0 +1,5 @@
+supply,30
+buy,10
+buy,13
+supply,17
+buy,10
```

Comment:
don't push csv files. delete this file from PR

Reply:
I deleted cvs files and main class.
I used those for seeing as it wrote the file.